.github/workflows/ci.yml

```diff
@@ -123,6 +123,72 @@ jobs:
       - run: make man
         working-directory: src/github.com/containerd/containerd
 
+  # Make sure binaries compile with other platforms
+  crossbuild:
+    name: Crossbuild Binaries
+    needs: [project, linters, protos, man]
+    runs-on: ubuntu-18.04
+    timeout-minutes: 10
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - goos: linux
+            goarch: arm64
+          - goos: linux
+            goarch: arm
+            goarm: "7"
+          - goos: freebsd
+            goarch: amd64
```

Comment:
I think amd64 is reasonable here as it's all I've tested with.  I'd like to test on arm64 too, but haven't had the chance to yet.

Reply:
It compiles at least :) Added.